test/e2e/tests/privacy/basic-functionality.spec.js

```diff
@@ -1,176 +0,0 @@
-        // otherwise the click has no effect.
-        await driver.waitForElementToStopMoving(
-          '[data-testid="privacy-settings-back-button"]',
-        );
-        await driver.clickElement(
-          '[data-testid="privacy-settings-back-button"]',
-        );
-
-        await driver.clickElementAndWaitToDisappear({
-          text: 'Done',
-          tag: 'button',
-        });
-        await driver.clickElement({
-          text: 'Next',
-          tag: 'button',
-        });
-
-        // Wait until the onboarding carousel has stopped moving
-        // otherwise the click has no effect.
-        await driver.waitForElementToStopMoving({
-          text: 'Done',
-          tag: 'button',
-        });
-        await driver.clickElementAndWaitToDisappear({
-          text: 'Done',
-          tag: 'button',
-        });
-
-        await driver.clickElement('[data-testid="network-display"]');
-
-        await driver.clickElement({ text: 'Ethereum Mainnet', tag: 'p' });
-
-        // Wait until network is fully switched and refresh tokens before asserting to mitigate flakiness
-        await driver.assertElementNotPresent('.loading-overlay');
-        await driver.clickElement(`[data-testid="import-token-button"]`);
-        await driver.clickElement('[data-testid="refreshList"]');
-
-        for (let i = 0; i < mockedEndpoints.length; i += 1) {
-          const requests = await mockedEndpoints[i].getSeenRequests();
-
-          assert.equal(
-            requests.length,
-            0,
-            `${mockedEndpoints[i]} should make requests after onboarding`,
-          );
-        }
-      },
-    );
-  });
-
-  it('should not prevent network requests to basic functionality endpoints when the basic functionality toggle is on', async function () {
-    await withFixtures(
-      {
-        fixtures: new FixtureBuilder({ onboarding: true }).build(),
-        ganacheOptions: defaultGanacheOptions,
-        title: this.test.fullTitle(),
-        testSpecificMock: mockApis,
-      },
-      async ({ driver, mockedEndpoint: mockedEndpoints }) => {
-        await importSRPOnboardingFlow({ driver });
-
-        await driver.clickElement({
-          text: 'Manage default privacy settings',
-          tag: 'button',
-        });
-        await driver.clickElement('[data-testid="category-item-General"]');
-        // Wait until the onboarding carousel has stopped moving
-        // otherwise the click has no effect.
-        await driver.waitForElementToStopMoving(
-          '[data-testid="category-back-button"]',
-        );
-        await driver.clickElement('[data-testid="category-back-button"]');
-        // Wait until the onboarding carousel has stopped moving
-        // otherwise the click has no effect.
-        await driver.waitForElementToStopMoving(
-          '[data-testid="privacy-settings-back-button"]',
-        );
-        await driver.clickElement(
-          '[data-testid="privacy-settings-back-button"]',
```

Comment:
all these steps are unnecessary, it's just navigating back and forth to different setting pages, but the correct settings are already set. The default settings are the ones we need for this spec so no other action is needed (compared to the previous test, where we did need to change the default settings to the ones we needed for that test)